src/py-opentimelineio/opentimelineio/url_utils.py

```diff
@@ -40,14 +40,15 @@ def url_from_filepath(fpath):
         )
 
 
-def filepath_from_url(urlstr):
+def filepath_from_url(urlstr, as_posix=True):
```

Comment:
Is the `as_posix` argument used anywhere?

Reply:
Ah, this can be removed I was playing around with ideas and forgot to remove this.